boards/PDM/Src/Io/Io_Efuse.c

```diff
@@ -0,0 +1,350 @@
+#include <assert.h>
+#include <math.h>
+#include <stdlib.h>
+#include "Io_Efuse.h"
+#include "configs/Io_EfuseConfig.h"
+
+#pragma GCC diagnostic ignored "-Wconversion"
+
+struct Efuse_Context
+{
+    // The SPI handle for the SPI device the E-Fuse is connected to
+    SPI_HandleTypeDef *efuse_spi_handle;
```

Comment:
The struct pointing to the spi handle already indicates that it pertains to the Efuse so we should rename it to `hspi`

```suggestion
    SPI_HandleTypeDef *hspi;
```